datafusion/execution/src/cache/list_files_cache.rs

```diff
@@ -21,9 +21,9 @@ use std::{
 };
 
 use datafusion_common::instant::Instant;
-use object_store::{path::Path, ObjectMeta};
+use object_store::{ObjectMeta, path::Path};
```

Comment:
This is unrelated, but was causing the `cargo fmt` CI job to fail.

Reply:
👍  @adriangb  seems to have fixed this in the recent commit to main
- https://github.com/apache/datafusion/pull/19102